src/PackageUrl.csproj

```diff
@@ -13,4 +13,8 @@
     <PackageOutputPath>../nuget</PackageOutputPath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Would System.Text.Json perhaps be a better alternative here instead so that dependencies don't have to pull in Newtonsoft?